source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,228 @@
 ---
 title: HTML coding style
-last_reviewed_on: 2016-02-03
+last_reviewed_on: 2019-05-24
 review_in: 6 months
 ---
 
 # <%= current_page.data.title %>
 
-## HTML5
+## Progressive enhancement
 
-* Always quote attribute values - the decision tree for remember when to
-  selectively quote is too much mental overhead, so just quote them
-* Always use well-formed HTML - implicitly closed elements such as
-  `<p>` cause unnecessary parsing overhead and can cause issues in
-  accessibility
+HTML should be written in such a way that conforms with our [browser support model](https://github.com/alphagov/govuk-frontend#browser-support).
 
-## Use [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) roles appropriately
+For example, if IE8 is a supported browser, and it doesn't support the `<video>`
+tag, you should ensure an appropriate fallback is in place. Example:
 
-### Guidance on using [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) landmarks
+```html
+<video width="320" height="240" controls>
+  <source src="movie.mp4" type="video/mp4">
+  <source src="movie.ogg" type="video/ogg">
+  Your browser does not support the video tag.
+</video>
+```
 
-When you use HTML5 sectioning elements it's a good idea to include
-the appropriate [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) landmark roles as well. The trick when you're
-using both HTML5 and [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) is to put the landmark on the sectioning
-element itself because this avoids the information being duplicated by
-screen readers that support both technologies.
+When using structual HTML5 elements, such as `<main>`, ensure that you include
+a shiv that allows them to be styled in legacy browsers:
```

Comment:
Since a shiv is a knife, can we use a different phrase please?

```suggestion
a shim / patch that allows them to be styled in legacy browsers:
```